app/javascript/mastodon/features/notifications_v2/components/notification_admin_report_note.tsx

```diff
@@ -0,0 +1,66 @@
+import { FormattedMessage } from 'react-intl';
+
+import classNames from 'classnames';
+
+import FlagIcon from '@/material-icons/400-24px/flag-fill.svg?react';
+import { Icon } from 'mastodon/components/icon';
+import { RelativeTimestamp } from 'mastodon/components/relative_timestamp';
+import type { NotificationGroupAdminReportNote } from 'mastodon/models/notification_group';
+import { useAppSelector } from 'mastodon/store';
+
+export const NotificationAdminReportNote: React.FC<{
+  notification: NotificationGroupAdminReportNote;
+  unread?: boolean;
+}> = ({ notification, notification: { reportNote }, unread }) => {
+  const account = useAppSelector((state) =>
+    state.accounts.get(notification.sampleAccountIds[0] ?? '0'),
+  );
+
+  if (!account) return null;
+
+  const domain = account.acct.split('@')[1];
+
+  const values = {
+    name: <bdi>{domain ?? `@${account.acct}`}</bdi>,
+    reportId: reportNote.report.id,
+  };
+
+  const message = (
+    <FormattedMessage
+      id='notification.admin.report_note'
+      defaultMessage='{name} added a report note to Report #{reportId}'
+      values={values}
+    />
+  );
+
+  return (
+    <a
+      href={`/admin/reports/${reportNote.report.id}#report_note_${reportNote.id}`}
```

Comment:
Note: this will jump you straight to that report note once #31529 lands, as that adds the `domId` stuff.